### DISCUSSION TITLE
Cameras running wz_mini_hacks unaccessible in Alexa App

### DISCUSSION THREAD

**Discussion by bjs-pdx** [gtxaspec/wz_mini_hacks#508] [General]
I haven't been able to thoroughly test this yet, but it seems that my cameras running wz_mini_hacks and connected via POE/ethernet with the Uctronics adapter are not accessible for viewing within the Alexa App.  The cameras just timeout.  It doesn't seem to happen with the couple of cameras I have running over WIFI.  

I'm going to try switching a camera from ethernet to WIFI later to see what happens.  Just wondered if anyone else has experienced this or could also test.

**Comment by Bob-AOL** [comment]
Same network for both WiFi and ethernet? And the cameras time out, or the app times out? All v3s with the same Wyze firmware version?

**Reply by bjs-pdx** [reply]
Discovered that if I reboot the cameras WITHOUT wz_mini_hacks they start working.  Then they work with or without it.  Very strange.  Not that important since I don't use Alexa much, but I did find it interesting.

**Reply by Bob-AOL** [reply]
Glad to hear you figured it out. Good info to post in case someone else encounters the same issue.
